src/allmydata/test/web/test_webish.py

```diff
@@ -30,7 +48,7 @@
 
 from ...webish import (
     TahoeLAFSRequest,
-    tahoe_lafs_site,
```

Comment:
Aaaaaaaaa, this was a class name??

Reply:
Fortunately just a factory function not a class (or ... I dunno ... a partial application of a class?)